tests/orbit_determination/spacecraft.rs

```diff
@@ -329,7 +329,7 @@ fn od_val_sc_srp_estimation(
     let setup = Propagator::dp78(
         sc_dynamics,
         IntegratorOptions::builder()
-            .max_step(1.minutes())
+            .max_step(30.seconds())
```

Comment:
Prevents appearance of NaN in state vector with the Dormand Prince integrator. It's also a divisor of one minute, so the propagator simply takes two steps forward of the same distance to reach the next measurement. This issue seems specific to DP78, it didn't show up with RK89.